hass_nabucasa/iot.py

```diff
@@ -170,11 +170,10 @@ async def _reconnect() -> None:
 
 
 @HANDLERS.register("remote_sni")
-async def async_handle_remote_sni(cloud, payload):
+async def async_handle_remote_sni(cloud: Cloud, payload):
     """Handle remote UI requests for cloud."""
-    caller_ip = payload["ip_address"]
-
-    await cloud.remote.handle_connection_requests(caller_ip)
+    # caller_ip = payload["ip_address"]
```

Comment:
```suggestion
```

Reply:
The reason I had left this comment is so we wouldn't forget it was there in the future.